updating/updating_a_cluster/updating_disconnected_cluster/images

```diff
@@ -0,0 +1 @@
+../images
```

Comment:
These lower level symlinks should actually be backing all the way up to the top-level and then down to the right dir there (3 levels up, then to images in this case) [1]. Not sure if this is a showstopper though - @kalexand-rh is this anything that breaks the customer portal build sometimes, or is this not critical?

[1] https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#directory-names

Reply:
Whoops, didn't know that! I can fix these all now, even if it's preferred and not required